chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -1298,50 +1298,57 @@ See  `description <https://www.geo.admin.ch/en/geo-services/geo-services/portray
 
 The service provides both *tiles* and *styles* that the customer can use.
 
-GetTile
-*******
+GetStyle
+********
 
-A tile request is in the following form:
+A current (latest version) style request is in the following form:
 
 ::
 
-    <Scheme>://<ServerName>/tiles/<LayerName>/<version>/<zoomlevel>/<x>/<y>.pbf
-
-example of one pbf tile:
+    <Scheme>://<ServerName>/styles/<layername>/style.json
 
-- https://vectortiles.geo.admin.ch/tiles/ch.swisstopo.leichte-basiskarte.vt/v1.0.0/7/67/44.pbf
+example of current maplibre styles of light base map and imagery base map:
 
-GetTileSets
-***********
+- https://vectortiles.geo.admin.ch/styles/ch.swisstopo.leichte-basiskarte.vt/style.json
+- https://vectortiles.geo.admin.ch/styles/ch.swisstopo.leichte-basiskarte-imagery.vt/style.json
 
-MBTiles for storing  tiled map data in SQLite databases for immediate or offline usage and for efficient transfer. A MBtileSet request is in the following form:
+A previous versioned style request is in the following form:
 
 ::
 
-    <Scheme>://<ServerName>/tiles/<LayerName>/<version>/<LayerName>.mbtiles
+    <Scheme>://<ServerName>/styles/<layername>/<version>/style.json
 
-example of the .mbtiles file:
 
-- https://vectortiles.geo.admin.ch/tiles/ch.swisstopo.leichte-basiskarte.vt/v1.0.0/ch.swisstopo.leichte-basiskarte.vt.mbtiles
+(available previous versions are: v1.1.0, v1.2.0, v1.3.0, v1.4.0, v1.5.0 and v1.6.0)
```

Comment:
This is just an educated guess after quickly scanning the  `/styles/<layername>/ directories in the bucket `vectortiles-prod-swisstopo` in`swisstopo-bgdi` account.
Is this correct or am I missinterpreting something here? @procrastinatio @rebert @ltclm ?

Reply:
@hansmannj 
Those versions are working and seem to be correct.
 please also update last example "example of tileset" (Line 1366) to newest version v2.0.0 (Sorry I cant manage to add a comment at the specific line :-))

Reply:
should be done now @deniers. It's in line 1373 now.

Reply:
Testlink: http://mf-chsdi3.dev.bgdi.ch/feature_BGDIINF_SB-2356_update_api_docu_vector_tiles/services/sdiservices.html#mapbox-vector-tiles